src/zappy_ai_src/ai_strategy_management.py

```diff
@@ -244,7 +244,7 @@ def __getPlayerCurrentLevel(self) -> int:
             where x is the level of the player
     """
     def __getPlayerMaxRange(self) -> int:
-        playerLevel = self.__playerCurrentLevel + 1
+        playerLevel = self.__playerCurrentLevel
```

Comment:
No the previous one was good because the level 1 player can see 0, 1, 2, 3 so 4 tiles.
(1 + 1)² = 4

Reply:
No level 1 can only see his tile